components/decimal/src/lib.rs

```diff
@@ -62,15 +62,15 @@
 //! use icu::locale::locale;
 //! use writeable::assert_writeable_eq;
 //!
-//! let fdf = DecimalFormatter::try_new(
+//! let df = DecimalFormatter::try_new(
```

Comment:
could you rename all `let df =` to `let formatter =`? I think we usually call these things `formatter`

Reply:
done